styles/styles.css

```diff
@@ -181,19 +192,8 @@ header {
     margin: 0 auto;
   }
 
-  main .section .default-content-wrapper {
```

Comment:
I was thinking to just repurpose these, dropping the .default-content-wrapper.

Reply:
we will remove it after we will agree on the final structure